elements/rh-button/rh-button.css

```diff
@@ -499,3 +484,24 @@ button[disabled] {
 :host(:disabled) button:after {
   --_border-color: transparent;
 }
+
+/******************************
+ *                            *
+ *     SECONDARY-DANGER       *
+ *                            *
+ ******************************/
+
+:host([variant="secondary" i][danger]) button {
```

Comment:
What does the `i` do here in the css?

Reply:
or is it just a typo?

Reply:
makes it case insensitive
https://developer.mozilla.org/en-US/docs/Web/CSS/Attribute_selectors#attrvalue_4